en-US/snippets/2018/Firefox100-part1.lang

```diff
@@ -0,0 +1,295 @@
+## NOTE: Reference bug 1463556; Deadline: June 11, 2018
```

Comment:
This means a handful of days once we're done with review. Is this date real?

Reply:
I meant July, :).